src/telegram2discord/From.ts

```diff
@@ -79,9 +79,16 @@ export function createFromObjFromChat(chat: Record<string, any>) {
  * @returns The display name
  */
 export function makeDisplayName(useFirstNameInsteadOfUsername: boolean, from: From) {
-	return R.ifElse<any, any, any>(
-		from => useFirstNameInsteadOfUsername || R.isNil(from.username),
-		R.prop("firstName"),
-		R.prop("username")
-	)(from);
+	if (useFirstNameInsteadOfUsername || !from.username) {
+		const prefix: string = from.lastName ? ` ${from.lastName}` : "";
+		return `${from.firstName}${prefix}`;
```

Comment:
```suggestion
		const suffix: string = from.lastName ? ` ${from.lastName}` : "";
		return `${from.firstName}${suffix}`;
```

it's actually a suffix in this context